src/Azure.Functions.Cli/Helpers/TargetFrameworkHelper.cs

```diff
@@ -5,7 +5,7 @@ namespace Azure.Functions.Cli.Helpers
 {
     public static class TargetFrameworkHelper
     {
-        private static readonly IEnumerable<string> supportedTargetFrameworks = new string[] { TargetFramework.net8, TargetFramework.net7, TargetFramework.net6, TargetFramework.net48 };
+        private static readonly IEnumerable<string> supportedTargetFrameworks = new string[] { TargetFramework.net9, TargetFramework.net8, TargetFramework.net7, TargetFramework.net6, TargetFramework.net48 };
```

Comment:
Let's also add tests for the relevant actions. The two that come to mind are "init" and "start." Additionally, we should consider checking the deployment-related actions as well.

Reply:
Added tests for func init and start